architecture/infrastructure_components/image_registry.adoc

```diff
@@ -54,3 +54,96 @@ repositories using credentials supplied by the user. This allows {product-title}
+images and hosted content on {product-title}. Following the move to the new
+registry, the existing registry will be available for a period of time.
+
+The new registry uses standard OAuth mechanisms for authentication,
+with the following methods:
+
+* *Web username and password.* This is the standard set of credentials you use
+to log in to resources such as `access.redhat.com`.
+* *Authentication token.* These tokens, generated by administrators, are non-user
+accounts that give systems the ability to authenticate against
+the container registry.
+
+You can use `docker login` with your credentials, username and password or authentication
+token, to access content on the new registry.
+
+All image streams are updated to point to the new registry. Because the
+new registry requires authentication for access, there is a new secret in the
+OpenShift namespace called `imagestreamsecret`.
+
+There are two places your credentials must exist:
+
+* *OpenShift namespace*. Your credentials must exist in the OpenShift
+namespace so that the image streams in the OpenShift namespace can import.
+* *Your host*. Your credentials must exist on your host because Kubernetes will
+use the credentials from your host when it goes to pull images.
+
+As long as you have the following items in place, you should be able to
+access the new registry.
+
+* You should see image import secret, `imagestreamsecret`,  in your OpenShift
+namespace. That secret should have credentials that will allow you to access
+the new registry.
+* All of your cluster nodes should have a `dockerconfigjson` that allows you to
+access the Red Hat registry.
+
+[[create-user-accounts]]
+=== Creating User accounts
+
+//Concept
+
+If you are a Red Hat customer with entitlements to Red Hat products, you have
+an account with applicable user credentials. These are the username and password
+that you use to log in to the Red Hat Customer Portal.
+
+If you do not have an account, you can acquire one for free by registering for
+one of the following options:
+
+* link:https://developers.redhat.com/[*Red Hat Developer Program*]. This account
+gives you access to developer tools and programs.
+* link:https://access.redhat.com/products/red-hat-enterprise-linux/evaluation[*30-day Trial Subscription*].
+This account gives you a 30-day trial subscription with access to select Red Hat
+software products.
+
+[[create-authentication-tokens]]
+=== Creating Authentication Tokens
+
+You must create token if your organization manages shared accounts.
+Administrators can create, view, and delete all tokens associated an
+organization.
+
+//[[Prerequisites]]
+*Prerequisites*
+
+* User credentials
+
+//[[Procedure]]
+*Procedure*
+
+To create a token in order complete a `docker login`:
+
+. Navigate to `registry.redhat.io`.
+. Log in with your RHN username and password.
+. Accept terms when prompted.
+.. If you are not immediately prompted to accept terms, you will be prompted
```

Comment:
If you have already accepted "terms", then you will not be prompted/need to accept them again.